src/main/java/calculator/NumberCalculator.java

```diff
@@ -0,0 +1,14 @@
+package calculator;
+
+import java.util.List;
+
+public class NumberCalculator {
+
+	public static final int DEFAULT_VALUE = 0;
```

Comment:
```suggestion
    public static final int DEFAULT_VALUE = 0;
```
ide의 indent 설정을 확인 부탁드립니다. 😄 
https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent